pkg/wfc3/lib/donoise.c

```diff
@@ -140,21 +140,22 @@ int *done        o: true if we actually did assign error array values
         /* Now square the readnoise */
         for (i = 0; i < NAMPS; i++)
 	     rn2[i] = rn2[i] * rn2[i];
-        
+
 	if (wf3->ncombine > 1) {
 	    trlwarn("NCOMBINE > 1 before the error array was initialized.");
 	}
 
         /* Now apply the initilalization for each AMP used */
         for (j = 0; j < ampy; j++) {
-        
-            /* This region corresponds to AMP_C, 
+
+            /* This region corresponds to AMP_C,
                 if it is even used for this observation. */
 	    /* Let's make sure we actually found a value for the gain
 	    **	and readnoise... */
 
-	    if (ampx > 0 && (gain[AMP_C] == 0. || rn2[AMP_C] == 0.)) { 
-                trlerror("No valid GAIN or READNOISE values to initialize ERR data.");
+	    if (ampx > 0 && (gain[AMP_C] == 0. || rn2[AMP_C] == 0.)) {
+                trlerror
+		  ("No valid GAIN or READNOISE values to initialize ERR data.");
```

Comment:
In this file the "trlerror"  is split from its message ("blah blah") _several_ times in this file.  Can we keep it together?

Lines 157, 178, 202, and 225.

Reply:
Thanks I'll fix them. On the bright side I'd never put a new line after a function call on purpose. My editor must have been having a moment!

Reply:
Someone used a code formatter in drizzlepac and ending getting lines like this:

```
here_is_some_python_code
[0]
here_is some_more_python_code
```
Noooooooo.

Reply:
Ha, no I haven't... 
Done